gravitee-am-management-api/gravitee-am-management-api-services/gravitee-am-management-api-services-purge/src/main/java/io/gravitee/am/management/services/purge/PurgeManager.java

```diff
@@ -87,42 +87,19 @@ public void purge(List<TableName> exclude) {
-                    break;
-                case authorization_codes:
-                    authorizationCodeRepository.purgeExpiredData().subscribe();
-                    break;
-                case refresh_tokens:
-                    refreshTokenRepository.purgeExpiredData().subscribe();
-                    break;
-                case request_objects:
-                    requestObjectRepository.purgeExpiredData().subscribe();
-                    break;
-                case scope_approvals:
-                    scopeApprovalRepository.purgeExpiredData().subscribe();
-                    break;
-                case login_attempts:
-                    loginAttemptRepository.purgeExpiredData().subscribe();
-                    break;
-                case uma_permission_ticket:
-                    permissionTicketRepository.purgeExpiredData().subscribe();
-                    break;
-                case auth_flow_ctx:
-                    authenticationFlowContextRepository.purgeExpiredData().subscribe();
-                    break;
-                case pushed_authorization_requests:
-                    pushedAuthorizationRequestRepository.purgeExpiredData().subscribe();
-                    break;
-                case ciba_auth_requests:
-                    cibaAuthRequestRepository.purgeExpiredData().subscribe();
-                    break;
-                case user_activities:
-                    userActivityRepository.purgeExpiredData().subscribe();
-                    break;
-                case devices:
-                    deviceRepository.purgeExpiredData().subscribe();
-                    break;
+                case access_tokens -> accessTokenRepository.purgeExpiredData().subscribe();
+                case authorization_codes -> authorizationCodeRepository.purgeExpiredData().subscribe();
+                case refresh_tokens -> refreshTokenRepository.purgeExpiredData().subscribe();
+                case request_objects -> requestObjectRepository.purgeExpiredData().subscribe();
+                case scope_approvals -> scopeApprovalRepository.purgeExpiredData().subscribe();
+                case login_attempts -> loginAttemptRepository.purgeExpiredData().subscribe();
+                case uma_permission_ticket -> permissionTicketRepository.purgeExpiredData().subscribe();
+                case auth_flow_ctx -> authenticationFlowContextRepository.purgeExpiredData().subscribe();
+                case pushed_authorization_requests -> pushedAuthorizationRequestRepository.purgeExpiredData().subscribe();
+                case ciba_auth_requests -> cibaAuthRequestRepository.purgeExpiredData().subscribe();
+                case user_activities -> userActivityRepository.purgeExpiredData().subscribe();
+                case devices -> deviceRepository.purgeExpiredData().subscribe();
+                default -> LOGGER.debug("Purging not implemented for table '{}'", toProcess);
```

Comment:
**remark**: this will not work for 4.0.x and 3.x as we are using JDK 11 and as far as know this syntax has been introduced after JDK 11. Remember to amend the mergify PR for these two versions before merging.